### PR TITLE
Add laurent story

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/_posts/2021-01-14-softozor-fusionauth.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+**Laurent:**  Softozor is a very little company made up by me, my brother and a few friends. Currently, we work together in our free time. We are mostly physicists and mathematicians. My brother and I work in software development companies. Our friends are teachers. One of the goals of the Softozor company is to produce educational video games. In order to achieve that goal, we need money, which we are trying to earn with software development projects. Currently, we are developing an ecommerce platform helping craftsmen to sell their products and make them visible in the regions they are living in. A craftsman can be a farmer, a caterer, a baker, a butcher, etc.
+
+I am the company's official leader. I organize the developer, architect, and product owner roles. I am the only one in the team to have the role of product owner. The roles of developer and architect are taken by other members of the team. Every architectural decision is discussed in the team so that we get as much insight as possible into the possible problems that we might face during the development. This way we also spread the knowledge of everyone across the whole team.
```
Comment (zadigus) [inline site/_posts/2021-01-14-softozor-fusionauth.md:24]: I wanted to say that I am playing the roles of developer, architect, and product owner. I thought that "energizing" a role would be correct English (that's the term that David Allen is using in his book "Holacracy").

File: site/_posts/2021-01-14-softozor-fusionauth.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Nowadays, I essentially develop in C++, C# (.Net / WPF), javascript (VueJs), and python (mainly to write acceptance tests).  
+
+> The ease of installation and the set of features were the most decisive motivations to use [FusionAuth].
+
+**Dan:** Is the craftsman site live? Can you share the URL? 
+
+**Laurent:**  Yes, the [craftsman site is live](https://www.ameising.ch), in a reduced form. In the near future, it will show our customers all the shops we are managing. Currently, no shop is shown yet but they will soon pop on the map that we see on the homepage. The platform is not a traditional e-shop. It manages multiple e-shops where each e-shop has multiple tenants. What you see today is just kind of our crowd-funding: we sell baskets of vegetables, meat, fruits, bread, etc. Also, the project you see on that URL only targets Switzerland, no other country. It is a local project focusing on Switzerland.
```
Comment (zadigus) [inline site/_posts/2021-01-14-softozor-fusionauth.md:32]: After review of the people who we build the "ameising" website for, it turns out we sell baskets as written here but they don't contain meat. We will of course sell meat over this platform, but not now in the crowd-funding baskets. So I would remove "meat, " from the sentence.

File: site/_posts/2021-01-14-softozor-fusionauth.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+We are also very happy to use the python SDK in our acceptance tests. We use it to fill up the users database with the necessary credentials to then interact with our system under test. 
+
+Both SDKs saved us a lot of time. 
+
+**Dan:** Any general feedback/areas to improve?
+
+**Laurent:**  I think it would be great if FusionAuth could produce [Pasetos](https://github.com/paragonie/paseto). I know everyone on the web uses JWTs, but those are very easy to handle in an insecure way. They are insecure by default, whereas Pasetos are secure by default. *[Ed note: We have an open feature request for [paseto support](https://github.com/FusionAuth/fusionauth-issues/issues/773).]* 
+
+For our use-cases with hasura, it would not bring anything right now (see [this issue](https://github.com/hasura/graphql-engine/issues/3205)) but I can imagine that they could be useful.
```
Comment (zadigus) [inline site/_posts/2021-01-14-softozor-fusionauth.md:94]: I'd reformulate the end of the sentence as "but I can imagine that they could be useful in the near future". Feel free to adapt or not.

File: site/_posts/2021-01-14-softozor-fusionauth.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+**Laurent:**  Our system is based on the 3-factor app pattern: realtime graphql API, reliable eventing, and async serverless. That is out-of-the-box hasura behavior. 
+
+Whenever some relevant data are changed in our database, some events are triggered, which results in calling some serverless functions (e.g. in our system we use OpenFaaS). In addition to that, we can complete the GraphQL API automatically generated by hasura with additional queries / mutations, which are known as "hasura actions". For example, the automatically generated API does not come out with a login mutation. Either you route the login request from your frontend to FusionAuth directly or you decide that you have a central API over which everything should go. 
+
+We chose the latter variant, as we find it clearer. In that case, we had to define a hasura login action. That amounts to telling hasura what the mutation looks like: what name, what input, what output, and then link that mutation with a serverless function. In that context, we are very happy to use the .Net SDK provided by FusionAuth! It has all the necessary REST client wrappers necessary for us to perform the operations we need on FusionAuth.
```
Comment (zadigus) [inline site/_posts/2021-01-14-softozor-fusionauth.md:84]: In the first sentence, I'd say "as we find it clearer and cleaner" or something like that. It is both clear and clean.

**Review by zadigus — CHANGES_REQUESTED** [review]
Voilà, I've reviewed the text you propose. For me the wording is perfectly fine, except for a few little details that I'd like you to adapt if you deem it relevant.

File: site/_posts/2021-01-14-softozor-fusionauth.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2021-01-14-softozor-fusionauth.md:24]: I hadn't heard that before, but am happy to use that language if that is more true to your intent.

**Review by zadigus — APPROVED** [review]
LGTM

File: site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+> Both [the .NET and python SDKs] saved us a lot of time. 
+
+**Dan:** Why did you choose FusionAuth over the competition?
+
+**Laurent:** We evaluated a few options: keycloak, keratin/authn, and FusionAuth. The ease of installation and the set of features were the most decisive motivations to use your software. Its documentation is outstanding, and that also helped a lot in choosing FusionAuth. 
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Laurent:** It saved a lot of time. If we agree that time is money, then it saved us a lot of money too. We could've implemented a reduced set of the features provided by FusionAuth, but that would've taken a lot of time. We could've chosen another solution than FusionAuth and implemented the missing features, but again that would've been a killer. I'd say it saved us something like one man-year of time.
```
Comment (dpolstra) [inline site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md:70]: We could've chosen another solution *other* than FusionAuth

File: site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+**Laurent:**  Softozor is a very little company made up by me, my brother and a few friends. Currently, we work together in our free time. We are mostly physicists and mathematicians. My brother and I work in software development companies. Our friends are teachers. One of the goals of the Softozor company is to produce educational video games. In order to achieve that goal, we need money, which we are trying to earn with software development projects. Currently, we are developing an ecommerce platform helping craftsmen to sell their products and make them visible in the regions they are living in. A craftsman can be a farmer, a caterer, a baker, a butcher, etc.
+
+I am the company's official leader. I energize the roles of developer, architect, and product owner. I am the only one in the team to have the role of product owner. The roles of developer and architect are taken by other members of the team. Every architectural decision is discussed in the team so that we get as much insight as possible into the possible problems that we might face during the development. This way we also spread the knowledge of everyone across the whole team.
```
Comment (dpolstra) [inline site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md:24]: and product owner > with an emphasis on product owner. ~I am the only one in the team to have the role of product owner~

File: site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+Nowadays, I essentially develop in C++, C# (.Net / WPF), javascript (VueJs), and python (mainly to write acceptance tests).  
+
+> The ease of installation and the set of features were the most decisive motivations to use [FusionAuth].
+
+**Dan:** Is the craftsman site live? Can you share the URL? 
+
+**Laurent:**  Yes, the [craftsman site is live](https://www.ameising.ch), in a reduced form. In the near future, it will show our customers all the shops we are managing. Currently, no shop is shown yet but they will soon pop on the map that we see on the homepage. The platform is not a traditional e-shop. It manages multiple e-shops where each e-shop has multiple tenants. What you see today is just kind of our crowd-funding: we sell baskets of vegetables, fruits, bread, etc. Also, the project you see on that URL only targets Switzerland, no other country. It is a local project focusing on Switzerland.
```
Comment (dpolstra) [inline site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md:32]: ~It is a local project focusing on Switzerland~

File: site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+**Laurent:** We evaluated a few options: keycloak, keratin/authn, and FusionAuth. The ease of installation and the set of features were the most decisive motivations to use your software. Its documentation is outstanding, and that also helped a lot in choosing FusionAuth. 
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Laurent:** It saved a lot of time. If we agree that time is money, then it saved us a lot of money too. We could've implemented a reduced set of the features provided by FusionAuth, but that would've taken a lot of time. We could've chosen another solution than FusionAuth and implemented the missing features, but again that would've been a killer. I'd say it saved us something like one man-year of time. 
+
+**Dan:** How do you run FusionAuth (k8s, standalone tomcat server, behind a proxy, etc)?
+
+**Laurent:** We run FusionAuth on a k8s cluster. It is completely hidden behind our hasura api. We write hasura actions that interact with FusionAuth.
```
Comment (dpolstra) [inline site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md:74]: k8s (Kubernetes) cluster

File: site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+**Laurent:** We run FusionAuth on a k8s cluster. It is completely hidden behind our hasura api. We write hasura actions that interact with FusionAuth.  
+
+> I'd say it saved us something like one man-year of time.
+
+**Dan:** Can you tell me a bit more about the hasura actions? 
+
+**Laurent:**  Our system is based on the 3-factor app pattern: realtime graphql API, reliable eventing, and async serverless. That is out-of-the-box hasura behavior. 
+
+Whenever some relevant data are changed in our database, some events are triggered, which results in calling some serverless functions (e.g. in our system we use OpenFaaS). In addition to that, we can complete the GraphQL API automatically generated by hasura with additional queries / mutations, which are known as "hasura actions". For example, the automatically generated API does not come out with a login mutation. Either you route the login request from your frontend to FusionAuth directly or you decide that you have a central API over which everything should go.
```
Comment (dpolstra) [inline site/_posts/2021-03-02-softozor-fusionauth-hasura-kubernetes.md:82]: When ~Whenever some~

**Pull request merged**